src/NServiceBus.Msmq.AcceptanceTests/When_using_empty_instance_mapping_file.cs

```diff
@@ -0,0 +1,116 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using System;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using NUnit.Framework;
+    using Settings;
+
+    public class When_using_empty_instance_mapping_file : NServiceBusAcceptanceTest
+    {
+        [SetUp]
+        public void SetupMappingFile()
+        {
+            // this can't be static because the conventions are setup in the NServiceBusAcceptanceTest base class
+            logicalEndpointName = Conventions.EndpointNamingConvention(typeof(ScaledOutReceiver));
+
+            // e.g. spelling error in endpoint:
+            File.WriteAllText(mappingFilePath,
+                $@"<endpoints>
+    <endpoint name=""{logicalEndpointName}"">
+    </endpoint>
+</endpoints>");
+        }
+
+        [TearDown]
+        public void DeleteMappingFile()
+        {
+            File.Delete(mappingFilePath);
+        }
+
+        [Test]
+        public async Task Should_send_messages_to_logical_endpoint_address()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<SenderWithEmptyMappingFile>(e => e.When(async c =>
+                {
+                    for (var i = 0; i < 10; i++)
+                    {
+                        await c.Send(new Message());
+                    }
+                }))
+                .WithEndpoint<ScaledOutReceiver>(e => e.CustomConfig(c => c.MakeInstanceUniquelyAddressable("1")))
+                .WithEndpoint<ScaledOutReceiver>(e => e.CustomConfig(c => c.MakeInstanceUniquelyAddressable("2")))
+                .Done(c => c.MessagesForInstance1 + c.MessagesForInstance2 >= 10)
+                .Run();
+
+            // it should send messages to the shared queue
+            Assert.That(context.MessagesForInstance1, Is.GreaterThanOrEqualTo(1));
+            Assert.That(context.MessagesForInstance2, Is.GreaterThanOrEqualTo(1));
+        }
+
+        static string mappingFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, nameof(When_using_empty_instance_mapping_file) + ".xml");
+        static string logicalEndpointName;
+
+        public class Context : ScenarioContext
+        {
+            public int MessagesForInstance1;
+            public int MessagesForInstance2;
+        }
+
+        public class SenderWithEmptyMappingFile : EndpointConfigurationBuilder
+        {
+            public SenderWithEmptyMappingFile()
+            {
+                EndpointSetup<DefaultServer>(c =>
+                {
+                    var routingSettings = c.UseTransport<MsmqTransport>().Routing();
+                    routingSettings.RouteToEndpoint(typeof(Message), logicalEndpointName);
+                    routingSettings.InstanceMappingFile().FilePath(mappingFilePath);
```

Comment:
@danielmarbach I talked quickly to @bording about the API and we agreed that we prefer having the configuration options on the same level rather than splitting it on different config "sections" (the other approach would be to have 3 overloads for `InstanceMappingFile`). Are you good with that?